loader/windows/icd_windows_hkr.c

```diff
@@ -17,8 +17,9 @@
  */
 
 #include "icd.h"
-#include "icd_windows_hkr.h"
 #include <windows.h>
+#include "icd_windows_hkr.h"
```

Comment:
This change can be reverted now, since icd_windows_khr.h properly includes Windows.h,